src/pages/event-attendees/event-attendees.ts

```diff
@@ -56,6 +57,25 @@ export class EventAttendeesPage {
     });
   }
 
+  public doRefresh(refresher) {
+    this.attendeesService.loadAttendees(this.event.id).subscribe(
```

Comment:
@niranjan94 `refresher.complete()` needs to be called after the async work. Thats why I couldn't make a common function. Please suggest how can be duplication avoided in this case?